scripts/typecheck_tests.py

```diff
@@ -60,9 +59,18 @@ def replace_with_clickable_location(error: str, abs_test_folder: Path) -> str:
         import distutils.spawn
 
         mypy_executable = distutils.spawn.find_executable('mypy')
-        completed = subprocess.run([mypy_executable, *mypy_options], env={'PYTHONPATH': str(tests_root)},
-                                   stdout=subprocess.PIPE, cwd=str(tests_root))
-        sorted_lines = sorted(completed.stdout.decode().splitlines())
+        completed = subprocess.run(
+            [mypy_executable, *mypy_options],
+            env={'PYTHONPATH': str(tests_root)},
+            stdout=subprocess.PIPE,
+            cwd=str(tests_root)
+        )
```

Comment:
Could be kept as-is maybe, but I think we need to adjust PYTHONPATH there or something similar.